File: Python/Lista 2/multiples.py
V = input().split()
X, Y = V
X, Y = int(X), int(Y)
if Y > X:
   if Y%X == 0:
      print('Sao Multiplos')
   else:
      print('Nao sao Multiplos')
elif Y < X:
   if X%Y == 0:
      print('Sao Multiplos')
   else:
      print('Nao sao Multiplos')

File: Java/Lista 2/multiples.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        int X = sc.nextInt();
        int Y = sc.nextInt();
        if (Y > X) {
            if (Y%X == 0) {
                System.out.println("Sao Multiplos");
            } else {
                 System.out.println("Nao sao Multiplos");
            }
        } else if(Y < X) {
            if (X%Y == 0) {
                System.out.println("Sao Multiplos");
            } else {
                System.out.println("Nao sao Multiplos");
            }
        }
    }
}